src/web-fls-quiz/wwwroot/scripts/apps/quiz/components/intro/intro.js

```diff
@@ -1,12 +1,13 @@
-﻿define([],
+﻿define('apps/quiz/components/intro/intro', [],
```

Comment:
Something needs to be done about the bundle maker. The former bundle didn't need these additions, obviously we could do without them.